processor/src/app.js

```diff
@@ -37,21 +37,6 @@ function createApp(repoDir = '/gitea-data/git/repositories') {
 
   app.stop = unwatch
 
-  app.use((req, res, next) => {
-    const origin = req.get('origin')
-    if (!origin) {
-      return next()
-    }
-    if (ALLOWED_CORS_DOMAINS.includes(origin)) {
-      res.header('Access-Control-Allow-Origin', origin)
-      res.header('Access-Control-Allow-Methods', 'GET,OPTIONS')
-      res.header('Access-Control-Allow-Headers', 'Content-Type')
-      res.header('Access-Control-Allow-Credentials', 'true')
-      return next()
-    }
-    return res.sendStatus(403)
-  })
-
```

Comment:
The env variable needs to be removed properly everywhere.

Reply:
Which env variable?

Reply:
ALLOWED_CORS_DOMAINS for the processor

Reply:
Resolved via 09f3f2b.